Automattic/wp-calypso
# Add site editor urls to admin links

Fixes https://github.com/Automattic/wp-calypso/issues/95242 `Customize my theme is pointing to the old Customize area instead of Global Styles.`

## Proposed Changes

This PR adds the site editor urls to be showed in the admin links section of the Help Center, the `Show me where to` section:

<img width="442" alt="image" src="https://github.com/user-attachments/assets/aff14288-d159-424e-947a-8ee32b9d5e89">


## Why are these changes being made?


The old link is outdated.

## Testing Instructions



1. Live link
2. Open Help Center
3. Search for `Custom css`
4. Click on the `Customize my site's theme` link
5. You should land in the site editor, and not in the customizer.
